debian/eos-media.install

```diff
@@ -0,0 +1,4 @@
+usr/share/eos-media/default_images
+usr/share/eos-media/desktop-background-es_GT.jpg
```

Comment:
It seems a little fragile to have to remember to list each localized background image in this install file.  I suppose the eventual solution should be to get this out of the core OS altogether and have the image builder provide a default background image if not using the global default, so this is good enough for now.